charts/v2.0/cray-nls/values.yaml

```diff
@@ -104,7 +104,7 @@ cray-service:
       name: "cray-nls"
       image:
         repository: artifactory.algol60.net/csm-docker/stable/cray-nls
-        tag: 0.9.18
+        tag: 0.9.19
```

Comment:
Can we actually bump this to 0.9.20 to include Jesse's change? I'm just about to merge the PR that does the version bump here: https://github.com/Cray-HPE/cray-nls/pull/152

Then I'll push the 0.9.20 tag to the cray-nls repo. I can let you know when that's done.

Reply:
I merged that PR and pushed the v0.9.20 tag. The build of the container image is occurring here: https://github.com/Cray-HPE/cray-nls/actions/runs/4086357442